.travis.yml

```diff
@@ -69,6 +69,14 @@ matrix:
       env: TOXENV=py34
     - python: "3.5"
       env: TOXENV=py35
+    - os: osx
+      osx_image: xcode7.3
+      sudo: required
+      language: generic
+      env: TOXENV=cover
+      before_install: "travis_retry brew install python augeas dialog"
```

Comment:
nit: Saw [this](https://docs.travis-ci.com/user/osx-ci-environment/#Homebrew). Maybe we should change this to
```
before_install:
    - travis_retry brew update
    - travis_retry brew install python augeas dialog
```